_layouts/product.html

```diff
@@ -54,7 +54,7 @@
 {% if r.link and diff > 0 %}
 {{r.link}}
 {% elsif page.changelogTemplate and diff > 0 %}
-{{page.changelogTemplate | replace: '__RELEASE_CYCLE__',r.releaseCycle  | replace: '__LATEST__',latestVersionNumber | replace: '__LATEST_SHORT_HAND__',r.latestShortHand | replace: '__CYCLE_SHORT_HAND__',r.cycleShortHand | replace: '__CODENAME__',r.codename}}
+{{page.changelogTemplate | replace: '__RELEASE_CYCLE__',r.releaseCycle  | replace: '__LATEST__',latestVersionNumber | replace: '__LATEST_SHORT_HAND__',r.latestShortHand | replace: '__CYCLE_SHORT_HAND__',r.cycleShortHand | replace: '__CODENAME__',r.codename | replace: '__LATEST_RELEASE_DATE__', r.latestReleaseDate}}
```

Comment:
Can we make this one multi-line ? i mean something like this 
```
{{
   page.changelogTemplate | replace: '__RELEASE_CYCLE__',
   replace: '__LATEST__',latestVersionNumber | replace: '__LATEST_SHORT_HAND__',
  .....
}}
```

Reply:
@captn3m0 ?

Reply:
also dont we need to check if r.latestShortHand exist before creating __LATEST_SHORT_HAND__ ?
samething for others too ( like r.cycleShortHand , r.latestReleaseDate , ... )

Reply:
It gets replaced with an empty string I think.